### PR TITLE
Feature/githubworkflow

### PR DESCRIPTION
**Pull request by EstherAlo**

### CODE REVIEW

File: .taskcat.yml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+tests:
+
+  # compute
+  ## lambda
+  test-lambda:
+    parameters:
+      LambdaRole: iam-for-empty-lambda
+      SubnetIds: subnet-0a1c19ce3ee6b2d50,subnet-0e2e55747c817abac
+      SecurityGroupIds: "sg-05fa01fdf5d0dd615"
+      SecretsManagerArn: arn:aws:secretsmanager:eu-west-2:891376909120:secret:terraform-20240222170217389100000001-yES23Q
```
Comment (dangital) [inline .taskcat.yml:24]: Will need updating to `SSMParameterArn`

File: .github/workflows/pr-to-main.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Validate CloudFormation templates
+        run: |
+          for f in $(find ./service_catalog_portfolios -name '*.yaml')
+          do
+            echo 
+            aws cloudformation validate-template --template-body file://$f
```
Comment (dangital) [inline .github/workflows/pr-to-main.yaml:33]: Pretty sure this block will need to go after the AWS creds setup

**Review by dangital — CHANGES_REQUESTED** [review]
Workflow failing, couple of changes needed

File: .taskcat.yml (hunk 1 above)
Comment (melvabout) [reply .taskcat.yml:24]: on it

File: .github/workflows/pr-to-main.yaml (hunk 2 above)
Comment (melvabout) [reply .github/workflows/pr-to-main.yaml:33]: on it

File: .taskcat.yml (hunk 1 above)
Comment (melvabout) [reply .taskcat.yml:24]: will need a we bit of terraform

**Review by dangital — APPROVED** [review]
LGTM!

**Pull request merged**